ARGO/src/Game.cpp

```diff
@@ -233,8 +288,16 @@ void Game::preRender()
 	glm::vec2 focusPoint = glm::vec2(0, 0);
 	for (auto& player : m_players)
 	{
-		focusPoint.x += static_cast<TransformComponent*>(player.getAllComps().at(COMPONENT_ID::TRANSFORM_ID))->getPos().x;
-		focusPoint.y += static_cast<TransformComponent*>(player.getAllComps().at(COMPONENT_ID::TRANSFORM_ID))->getPos().y;
+		TransformComponent* transformComp = static_cast<TransformComponent*>(player.getAllComps().at(COMPONENT_ID::TRANSFORM_ID));
+		if (transformComp)
+		{
+			focusPoint.x += transformComp->getPos().x;
```

Comment:
why not just have:
focusPoint += transformComp->getPos();

Reply:
What?? That's what it is...